solr/src/search.coffee

```diff
@@ -738,6 +741,22 @@ body_quicklinks = () ->
       return [[input,tags,add_quicklinks(depart)]]
   }
 
+should_exclude_from_quicklinks = (title, doc) ->
+  # if a doc contains a quick_links field (which is an array)
+  # and if that array contains a string of a format "title: 0"
+  # (where title equals the first argument of the function,
+  # this quick link should not be shown on the page
+  rules = doc.quick_links || []
+
+  for rule in rules
+    if rule == 'none'
```

Comment:
this relies on the current (possibly incorrect) behaviour of solr (see PR description for details)